README.md

```diff
@@ -14,7 +14,6 @@ steps:
           path: path/to/deploy/code
           s3_bucket: deploybucket
           s3_key: deploy/key/name
-          region: ap-southeast-2
```

Comment:
It looks like the code uses `region`. I don't think we should remove this from the example.

If the linter is complaining, we'll need to add `region` as a parameter in the `plugin.yml` file.